python/README.md

```diff
@@ -39,7 +39,7 @@ After you git-clone this GitHub repository and ensure that `numpy` is installed,
 ```bash
 pip install "flatbuffers>=1.8.0"          # for the flatbuffers runtime (with Numpy)
 cd python                                 # only implementation so far is in Python
-python setup.py install                   # to use it outside of this directory
+python3 setup.py install                  # to use it outside of this directory
```

Comment:
Python 3 is not required, though these days, the generic `python` command ought to be Python 3 anyway. If rather the documentation just say "`python`".

Reply:
I run `python setup.py install` and it is broken for python2. If we are expected to use python3, I think this change would be good.

Reply:
That's weird. This was entirely written in Python 2 and not touched since then. (I didn't start using Python 3 regularly until rather late.) Python 3 is what we should highly recommend, but I'll need to go back and make sure that it works in Python 2 for other stragglers—someday. Python 2 is an extremely low priority.

Still, I think the command should be "python", not "python3", since many systems don't have a command named "python3" and for mainstream ones, the "python" command is Python 3. Maybe include a note that it's "currently broken for Python 2." That would be enough of a hint.

Reply:
Unless you force it, `python` should never be `python3`  in the main system unless some alternatives is set, Debian or RH base until PEP 394 is changed. In other systems, maybe. However, it is always okay to use in a virtual environment or conda, so `python` is still always the correct thing to write for a package that supports both.